tensorflow/compiler/xla/BUILD

```diff
@@ -963,7 +964,7 @@ cc_library(
         "debug_options_parsers.h",
     ],
     hdrs = ["debug_options_flags.h"],
-    copts = tf_copts(),
+    copts = if_enable_acl(["-DXLA_CPU_USE_ACL=1", "-fexceptions"]),
```

Comment:
Why this file in particular? Also do you know what `tf_copts()` was previously doing? Also why are we enabling exceptions? Is it possible to provide some context on what ACL is?

Reply:
Thanks for the quick review, @cheshire! ACL is [Arm Compute Library](https://github.com/ARM-software/ComputeLibrary), Arm's library of fast ML primitives. AWS made XLA CPU aarch64 backend call it a while ago https://github.com/tensorflow/tensorflow/pull/55534 (reviewed by @d0k).

> do you know what tf_copts() was previously doing?

`tf_copts` wasn't previously there (see #55534). It was [added later](https://github.com/tensorflow/tensorflow/pull/56715/) to fix [some aarch64 build issues](https://github.com/tensorflow/tensorflow/issues/56673). The inclusion of `tf_copts` breaks x86 oneDNN OpenMP build (`--config=mkl`) because of the flags x86 defines in `tf_copts`. So we are removing `tf_copts` since it wasn't originally there in the first place. (We plan to cherry-pick this into TF 2.10.)

> Why this file in particular?

@snadampal Could you please help answer why this file needs the flag? Thank you!

Reply:
Hi @cheshire , recently we added Arm Compute Library (ACL) support to xla cpu backend. The feature is guarded with "XLA_CPU_USE_ACL" flag which is part of tf_copts(). But adding the entire tf_copts() looks like bringing lot of additional flags which are breaking the x86 openmp builds. This commit is to replace tf_copt() with acl specific defines for these targets.

https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/xla/debug_options_flags.cc#L51
```
#ifdef XLA_CPU_USE_ACL
  opts.set_xla_cpu_use_acl(true);
#endif
```

Reply:
Why the need for exception support?

Reply:
@cheshire Great catch!
@snadampal The file doesn't really interact with ACL. (It just works on TF internal debug option settings.) I don't think we need `-fexceptions` here. What do you think?

Reply:
probably you are right as this is only a feature setting. Let me check once before we remove it.